Bottom: 55ae3af7c9cbf674722af74fd46c0b7546c170be
Top:    98389b9d566be843d09c21e9298957dd6371b00f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-05-05 19:01:42 +0100

Disable polygon debug gui


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index f6bf546..e8ef1d8 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1480,8 +1480,10 @@ ghid_build_pcb_top_window (void)
 
   gtk_tree_view_append_column (GTK_TREE_VIEW (ghidgui->polygon_debug_tv), column);
 
+#if 0
   gtk_box_pack_end (GTK_BOX (hbox_middle),
                     ghidgui->polygon_debug_tv, TRUE, TRUE, 0);
+#endif
 
   g_signal_connect (G_OBJECT (gtk_tree_view_get_selection (GTK_TREE_VIEW (ghidgui->polygon_debug_tv))),
                     "changed",
